common/src/main/java/muramasa/gregtech/data/Materials.java

```diff
@@ -620,19 +617,17 @@ public class Materials {
     public static Material Creosote = AntimatterAPI.register(Material.class, new Material(GTIRef.ID, "creosote", 0x804000, NONE));
     public static Material Diesel = AntimatterAPI.register(Material.class, new Material(GTIRef.ID, "diesel", 0xffff00, NONE));
     public static Material FishOil = AntimatterAPI.register(Material.class, new Material(GTIRef.ID, "fish_oil", 0xffc400, NONE));
-    public static Material HeavyFuel = AntimatterAPI.register(Material.class, new Material(GTIRef.ID, "heavy_fuel", 0xffff00, NONE));
-    public static Material NitroFuel = AntimatterAPI.register(Material.class, new Material(GTIRef.ID, "cetane_boosted_diesel", 0xc8ff00, NONE));
-    public static Material LightFuel = AntimatterAPI.register(Material.class, new Material(GTIRef.ID, "light_fuel", 0xffff00, NONE));
+    public static Material HeavyFuel = AntimatterAPI.register(Material.class, new Material(GTIRef.ID, "heavy_diesel", 0xffff00, NONE));
+    public static Material CetaneBoostedDiesel = AntimatterAPI.register(Material.class, new Material(GTIRef.ID, "cetane_boosted_diesel", 0xc8ff00, NONE));
+    public static Material LightFuel = AntimatterAPI.register(Material.class, new Material(GTIRef.ID, "light_diesel", 0xffff00, NONE));
```

Comment:
the merge change the names of light and heavy fuel to light and heavy diesel, need to change it back